src/simpleComponent.js

```diff
@@ -41,7 +41,7 @@ export default function(classNameBuilder) {
                 return { bemBlock : block };
         }
     }, {
-        displayName : 'Bem',
+        displayName : 'b:Bem',
```

Comment:
that's it true — `<Bem>` can be `b:` and `e:` too — i suggest to leave `Bem` here